app/assets/javascripts/discourse/tests/acceptance/tag-groups-test.js

```diff
@@ -36,3 +49,26 @@ test("tag groups can be saved and deleted", async (assert) => {
   await click(".tag-chooser .choice:first");
   assert.ok(!find(".tag-group-content .btn.btn-danger")[0].disabled);
 });
+
+QUnit.test(
+  "tag groups can have multiple groups added to them",
+  async (assert) => {
+    const tags = selectKit(".tag-chooser");
+    const groups = selectKit(".group-chooser");
+
+    await visit("/tag_groups");
+    await click(".content-list .btn");
+
+    await fillIn(".tag-group-content h1 input", "test tag group");
+    await tags.expand();
+    await tags.selectRowByValue("monkey");
+
+    await click("#private-permission");
+    assert.ok(find(".tag-group-content .btn.btn-default")[0].disabled);
```

Comment:
```suggestion
    assert.ok(find(".tag-group-content .btn.btn-default:disabled").length);
```
This is a huge nitpick but the query selector can find the disabled element which is slightly safer code than calling `[0]` which might not exist. I know it's a test, but still.